examples/example_stacked.py

```diff
@@ -0,0 +1,75 @@
+"""
+======================
+Stacked NUFFT Operator
+======================
+
+Example of Stacked NUFFT trajectory operator.
+
+This examples show how to use the Stacked NUFFT operator to acquire and reconstruct data
+in kspace where the sampling of pattern is a stack of non cartesian trajectory.
+Here a stack of spiral is used as a demonstration.
+
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+from mrinufft import display_2D_trajectory
+
+
+# %%
+# Data Generation
+# ===============
+# For realistic 3D images we will use the brainweb dataset.
+# installable using ``pip install brainweb``
+
+from brainweb_dl import get_mri
+
+mri_data = get_mri(0, "T1").T
+fig, ax = plt.subplots(1, 3)
+ax[0].imshow(mri_data[90, :, :])
+ax[1].imshow(mri_data[:, 108, :])
+ax[2].imshow(mri_data[:, :, 90])
```

Comment:
It would be nice to display those slices using conventional directions and colormap maybe ?